tripl/tripl.py

```diff
@@ -715,27 +724,26 @@ def pull_many(self, pull_expr, eids_or_pattern, sort_by=None, sort_desc=True):
         eids = self.match(eids_or_pattern) if isinstance(eids_or_pattern, dict) else eids_or_pattern
         results = (self.pull(pull_expr, eid, _seen_entities=_seen_entities) for eid in eids)
         if sort_by:
-            results = sorted(results, key = lambda x: x[sort_by])
+            results = sorted(results, key=lambda x: x[sort_by])
         if not sort_desc:
-            results = results.reversed()
+            results = reversed(results)
```

Comment:
This might be a Python 3 change: `results` has no field `reversed`, so I changed this to use the `reversed` builtin function instead.